administration/transport-security.md

```diff
@@ -171,3 +172,41 @@ Fluent Bit supports [TLS server name indication](https://en.wikipedia.org/wiki/S
     tls.ca_file /etc/certs/fluent.crt
     tls.vhost   fluent.example.com
 ```
+
+### Verify subjectAltName
+
+When extracting the X509v3 Subject Alternative Name from a certificate:
```

Comment:
I maybe would reword this a bit, using `Then` as well is not ideal.

Something like?
```suggestion.
By default, TLS verification of hostnames is not done automatically.
As an example, we can extract the X509v3 Subject Alternative Name from a certificate:
```